tensorflow/tools/dockerfiles/dockerfiles/devel-gpu.Dockerfile

```diff
@@ -38,6 +38,7 @@ ARG LIBNVINFER_MAJOR_VERSION=7
 SHELL ["/bin/bash", "-c"]
 RUN apt-get update && apt-get install -y --no-install-recommends \
         build-essential \
+        clang-format \
```

Comment:
We could install `clang-format-10` if we prefer `dry-run` but we always need to sync version with the CI Sanity VM